SONIC-DEV/sonic-buildimage
# [platform/cel]: Add bmc watchdog service

**- What I did**
- Add service to enable CPU watchdog on BMC
- Add a timer to send the hearhbeat request to BMC watchdog service

**- How I did it**
- Add bmc_wdt.sh for send a request to BMC RESTful API
- Add bmc_wdt.service for enable CPU watchdog on BMC for 60 sec 
- Add bmc_wdt.timer for run bmc_wdt.service every 20 sec 

**- How to verify it**
- Run command `systemctl status bmc_wdt.service` on SONIC to check service status
- Run command `tail /var/log/messages` on BMC to view watchdog enable/disable log
